server/src/main/java/com/paypal/selion/proxy/SeLionSauceProxy.java

```diff
@@ -23,71 +23,91 @@
 import java.util.logging.Level;
 
 import com.paypal.selion.grid.servlets.SauceServlet;
+import com.paypal.selion.utils.SauceConfigReader;
 import com.paypal.selion.utils.SauceLabsRestApi;
 
 import org.apache.commons.lang.StringUtils;
 import org.openqa.grid.common.RegistrationRequest;
 import org.openqa.grid.common.exception.GridConfigurationException;
 import org.openqa.grid.common.exception.GridException;
-import org.openqa.grid.internal.BaseRemoteProxy;
 import org.openqa.grid.internal.Registry;
 import org.openqa.grid.internal.TestSession;
+import org.openqa.grid.selenium.proxy.DefaultRemoteProxy;
 
 import com.paypal.selion.logging.SeLionGridLogger;
 
 /**
- * This customized version of {@link BaseRemoteProxy} basically helps redirect all traffic to the SauceLabs cloud.
+ * This customized version of {@link DefaultRemoteProxy} basically helps redirect all traffic to the SauceLabs cloud.
  */
-public class SeLionSauceProxy extends BaseRemoteProxy {
+public class SeLionSauceProxy extends DefaultRemoteProxy {
 
     private static final SeLionGridLogger LOGGER = SeLionGridLogger.getLogger(SeLionSauceProxy.class);
+
+    private static final String DEFAULT_TO_USERNAME_TOKEN = "$username";
+    private static final String PARENT_TUNNEL_CAPABILITY = "parent-tunnel";
```

Comment:
Would be nice if we put all these saucelabs desired capabilites Key strings in a seperate common file as consts or enums but its only spread across 2 files i think its okay.  But if we start spreading the need for accessing these desiredcaps key strings in more than 2 files we should move them to a common file as constants or enum.